### DISCUSSION TITLE
Maximize window rule

### DISCUSSION THREAD

**Discussion by raffaelemancuso** [hyprwm/Hyprland#1268] [Q&A]
Is there a maximize window rule?

I can only find a fullscreen one

**Comment by vaxerski** [comment]
https://wiki.hyprland.org/Configuring/Dispatchers/#list-of-dispatchers

![image](https://user-images.githubusercontent.com/43317083/209136680-b3e7fbe7-04e0-4fef-8d27-05174380d7d8.png)


blind

**Reply by ANGADSINGHT** [reply]
i love how ironic it is you said blind but you are blind yourself

**Comment by raffaelemancuso** [comment]
This is a dispatcher. I need a window rule https://wiki.hyprland.org/Configuring/Window-Rules/#rules

**Reply by vaxerski** [reply]
ah right, no there is no such rule

**Comment by kaunkrishna** [comment]
this is how i did it in my new dotfiles https://github.com/kaunkrishna/dotfiles (ik im a newbiee)
inside hypr/modules/keybinds.conf, from line 26 to 28 (fro now)

`# Fullscreen
bind = $mainMod, F, fullscreen, 1
bind = $mainMod SHIFT, F, fullscreen, 0`

the second one make it completely fullscreen no border or gaps

if this helped you, you can mark it as accepted 🙂
